### ISSUE TITLE
Expose `prefersEphemeralWebBrowserSession`

### ISSUE THREAD

**Issue by gersonmdesouza** [braintree/braintree_ios#1297]
### Is your feature request related to a problem? Please describe.

Hello,

We recently started offering Local Payment Methods to our users and we can see a gap on payment acceptance rate comparing Braintree to the old provider that we were using. We added tracking and logging to every place that we could and we noticed that iOS users are dropping mainly because of Apple's consent popup asking the user to share their credentials with paypal.com website. This is causing user misunderstanding what was the selected payment method, first the user selected iDeal or P24, but now the app is showing paypal.com message, so the user is canceling the flow.

Is there any way to control this field and stop sharing user credentials with the web browser session?
https://github.com/braintree/braintree_ios/blob/main/Sources/BraintreeCore/BTWebAuthenticationSession.swift#L9

### Describe the solution you'd like.

As a software developer and Braintree's client, I want to control whenever the user shares their credentials with web browser session during Local Payment Method flow, in order to increase payment acceptance rate and decrease misunderstandings.

**Comment by KunJeongPark** [comment]
Thank you for letting us know about this concern.
I will check with product and get back to you as soon as possible to let you know if this is possible.

**Comment by stechiu** [comment]
Hi @gersonmdesouza, we merged in changes this week to run an experiment to determine whether to enable/disable the WAS popup within the web fallback flow. We're waiting on changes from Billing team to be deployed before we can test these changes. Thanks for your patience.

**Comment by stechiu** [comment]
@gersonmdesouza Update - Billing team has deployed changes to prod and the experiment is `on`. These changes were merged into the latest `v6.34.0`. You can update to this version and test if the WAS popup no longer shows. Let us know if you're still having issues

**Comment by stechiu** [comment]
The solution I talked about earlier is for PayPal Checkout and you might continue seeing the WAS popup. I've reopened the issue.
